ui/app/templates/components/token-expire-warning.hbs

```diff
@@ -1,16 +1,16 @@
 {{#if (and this.showWarning (is-after (now interval=1000) @expirationDate))}}
-  <div class="token-expire-warning">
-    <AlertBanner
-      @type="danger"
-      @message="Your auth token expired on
-        {{date-format @expirationDate 'MMMM do yyyy, h:mm:ss a'}}
-         . You will need to re-authenticate."
-    >
-      <LinkTo @route="vault.cluster.logout" class="button link">
-        Reauthenticate
-      </LinkTo>
-    </AlertBanner>
-  </div>
+  <Hds::Alert @type="page" @color="critical" as |A|>
+    <A.Title>Error</A.Title>
+    <A.Description>
+      Your auth token expired on
+      {{date-format @expirationDate "MMMM do yyyy, h:mm:ss a"}}. You will need to re-authenticate.
+    </A.Description>
+    <A.Link::Standalone @icon="arrow-right" @iconPosition="trailing" @text="Reauthenticate" @route="vault.cluster.logout" />
+  </Hds::Alert>
 {{else}}
-  {{yield}}
+  <section class="section">
```

Comment:
Moved this here so the banner would span the whole UI width

![image](https://github.com/hashicorp/vault/assets/68122737/95b50604-17fe-4119-b59e-b16be54dffbe)